config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml

```diff
@@ -30,7 +30,7 @@ presubmits:
           limits:
             cpu: "2"
             memory: "6Gi"
-  - name: pull-kueue-test-integration-main
+  - name: pull-kueue-test-integration-required-main
```

Comment:
Please align the naming with https://github.com/kubernetes-sigs/kueue/pull/5068

Reply:
Done